src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -212,6 +218,11 @@ export default App.extend(extend({
   onCancelNewComment() {
     this.showNewCommentForm();
   },
+  onRemoveComment(model) {
+    this.action.removeComment(model);
+
+    Radio.request('ws', 'unsubscribe', model);
+  },
```

Comment:
Do we need `modal.destroy()` here? I don't think we do. It already happens in the view.

But the `onRemoveAttachment()` function in this file has it, which makes me unsure.